generation/2d_diffusion_autoencoder/2d_diffusion_autoencoder_tutorial.ipynb

```diff
@@ -0,0 +1,875 @@
+{
```

Comment:
<p>Perhaps move this description above the plot cell to let users more easier to see, what do you think?</p><p>BTW, please also include a remove temp dir cell as other tutorials, thanks.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1871/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Reply:
<p>Done</p>